src/nni_manager/common/manager.ts

```diff
@@ -36,6 +36,7 @@ interface ExperimentParams {
     trainingServicePlatform: string;
     multiPhase?: boolean;
     multiThread?: boolean;
+    versionCheck?: boolean;
```

Comment:
Is it necessary to add a swith in config file for version check?

Reply:
we will discuss this feature offline.

Reply:
change this param to be "debug" as discussed offline with @yds05